library/components/header/stream-picker.scss

```diff
@@ -88,6 +95,9 @@
       margin-right: 5px;
       order: 1;
     }
+    .stream-status {
```

Comment:
Maybe it's time to introduce a `type` props with `menu-item` or `dropdown`  instead of having display none?

Or even splitting in two components?

Reply:
I vote for splitting. 

In that case, just to align the display rules:

- Final status should not be displayed (neither in header nor in dropdown item)
- Recommended Stream should not be displayed, **only in header**, but it's ok to display it in the dropdown item